dataplane/standalone/apigen/docparser/docparser.go

```diff
@@ -23,6 +24,29 @@ import (
 	"strings"
 )
 
+// Info contains all the info parsed from the doxygen.
+type Info struct {
```

Comment:
SAIInfo to clarify what info is contained?
Another option is to reename the package saidocparser